From 6b9068e118daeaeb5037e422404ff356f277caff Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Mon, 9 Jul 2018 19:14:16 +0300
Subject: [PATCH 413/741] staging: fsl-dpaa2/eth: Enable debugfs support by
 default

commit 35bf49d738430e3cc93b182045b07975e4d33670 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

The debugfs counters of the DPAA2 Ethernet driver are useful in
many occasions and bring no runtime overhead, so it makes sense
to enable them by default.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/staging/fsl-dpaa2/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl-dpaa2/Kconfig b/drivers/staging/fsl-dpaa2/Kconfig
index b29b35c4490f..28cd595e97a2 100644
--- a/drivers/staging/fsl-dpaa2/Kconfig
+++ b/drivers/staging/fsl-dpaa2/Kconfig
@@ -33,7 +33,7 @@ config FSL_DPAA2_ETH_USE_ERR_QUEUE
 config FSL_DPAA2_ETH_DEBUGFS
 	depends on DEBUG_FS
 	bool "Enable debugfs support"
-	default n
+	default y
 	---help---
 	  Enable advanced statistics through debugfs interface.
 
-- 
2.17.1

